### PR TITLE
Adding hylafax printers

### PR DESCRIPTION
**Pull request by SNsir**

### CODE REVIEW

File: galite-core/src/main/kotlin/org/kopi/galite/util/PlatformFileWriter.kt [hunk 1]
```diff
@@ -15,9 +15,93 @@
...
+
+/**
+ * An class that simplifies file writing
+ */
 class PlatformFileWriter {
 
-}
+  // ----------------------------------------------------------------------
+  // CONSTRUCTOR
+  // ----------------------------------------------------------------------
```
Comment (ZMarouani) [inline galite-core/src/main/kotlin/org/kopi/galite/util/PlatformFileWriter.kt:32]: Constructors should be like this : 
class PlatformFileWriter(val dataStream: OutputStream,
                         val encoding: String,
                         val lineSeparator: String) {
  /**
   * Sets the number of copy to print
   */
  constructor(file: File,
              encoding: String,
              lineSeparator: String) : this(FileOutputStream(file) , encoding , lineSeparator  ) 
  
  constructor(fileName: String,
              encoding: String,
              lineSeparator: String) : this(File(fileName), encoding, lineSeparator)

File: galite-core/src/main/kotlin/org/kopi/galite/util/PlatformFileWriter.kt [hunk 2]
```diff
@@ -15,9 +15,93 @@
...
+    dataStream.close()
+  }
+
+  // ----------------------------------------------------------------------
+  // DATA MEMBERS
+  // ----------------------------------------------------------------------
+
+  private val encoding: String
+  private val lineSeparator: String
+  private val dataStream: OutputStream
```
Comment (ZMarouani) [inline galite-core/src/main/kotlin/org/kopi/galite/util/PlatformFileWriter.kt:106]: Delete

**Review by ZMarouani — CHANGES_REQUESTED** [review]

File: galite-core/src/main/kotlin/org/kopi/galite/form/VTextField.kt [hunk 3]
```diff
@@ -18,6 +18,9 @@
...
 package org.kopi.galite.form
 
-class VTextField(width: Int, height: Int, visibleHeight: Int, convert: Int, styled: Boolean): VStringField (width, height, visibleHeight, convert, styled)
-{
+/**
+ * This class implements multi-line text fields.
+ */
+class VTextField (width: Int, height: Int, visibleHeight: Int, convert: Int, styled: Boolean) : VStringField(width, height, visibleHeight, convert, styled) {
 }
+
```
[the remark this answers is not in the document]
Comment (SNsir) [reply galite-core/src/main/kotlin/org/kopi/galite/form/VTextField.kt:26]: yes ignore it, we will not implement it

File: galite-core/src/main/kotlin/org/kopi/galite/util/Fax.kt [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+ *
+ * $Id: Fax.java 34961 2016-11-04 17:20:49Z hacheni $
+ */
```
[the remark this answers is not in the document]
Comment (SNsir) [reply galite-core/src/main/kotlin/org/kopi/galite/util/Fax.kt:18]: already done in PR#92 , ignore it here

**Review by ZMarouani — APPROVED** [review]

**Pull request merged**